linux_docker_resources/Dockerfile

```diff
@@ -122,6 +122,16 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   uncrustify \
   yamllint
 
+# Configure proposed source, and install python3-flake8 from it
+RUN if test \( ${UBUNTU_DISTRO} = noble \); then\
+    printf "deb http://archive.ubuntu.com/ubuntu/ "$(lsb_release -cs)"-proposed restricted main multiverse universe" >> /etc/apt/sources.list.d/ubuntu-$(lsb_release -cs)-proposed.list && \
+    printf "Package: *\nPin: release a="$(lsb_release -cs)"-proposed\nPin-Priority: -10" >> /etc/apt/preferences.d/proposed-updates && \
+    cat /etc/apt/preferences.d/proposed-updates && \
```

Comment:
Is this just for visibility / debug purposes or does it serve some other intent?

Reply:
Debugging going to remove it. Had some issues with `echo -e` adding the `-e` character because that's a specific attribute from bash.